suite-common/connect-popup/src/connectPopupThunks.ts

```diff
@@ -7,7 +7,6 @@ import TrezorConnect, { CallMethodParams, CallMethodResponse } from '@trezor/con
 import { TypedError, serializeError } from '@trezor/connect/src/constants/errors';
 import { MethodPermission } from '@trezor/connect/src/core/AbstractMethod';
 import { DEEPLINK_VERSION } from '@trezor/connect/src/data/version';
-import { EventType, analytics } from '@trezor/suite-analytics';
```

Comment:
It was not included in package.json and tsconfig? 🤔

Reply:
Nope 😃